src/main/java/fr/socrates/domain/candidate/CandidateService.java

```diff
@@ -6,7 +6,7 @@
 import java.util.List;
 import java.util.Optional;
 public interface CandidateService {
-    boolean add(Candidate candidate);
+    boolean add(Candidate candidate) throws CandidatePersisteDataException, CandidateExistingException;
```

Comment:
as you handle errors with exceptions now, you should remove the boolean returned.
So the method could be `void add(Candidate candidate) throws CandidatePersisteDataException, CandidateExistingException;`

Reply:
done